ui/src/components/playPage/common/Badge.css

```
@@ -0,0 +1,168 @@
+  margin-top: 0.25rem;
+  z-index: 30;
+  transition: all 0.5s ease-in-out;
+}
+
+.action-display-box {
+  padding: 0.25rem 0.5rem;
+  border-radius: 0.375rem;
+  box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05);
+  border-width: 1px;
+  border-style: solid;
+  border-opacity: 0.8;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)